### ISSUE TITLE
Bug 59395701: [React Native Gallery Preview]: Tooltip is not defined to expand/collapse button

### ISSUE THREAD

**Issue by vineethkuttan** [microsoft/react-native-gallery#680]
Build Details:
OS Version: 24H2 
OS Build: 26100.5074
App: React Native Gallery Preview
App Version:1.0.18.0
AT Used: Keyboard

Repro-Steps:
Launch the React Native Gallery-Preview app.
Now navigate or hover on 'Expand/Collapse' button and observe the issue.
Actual Result:
Tooltip is not defined to expand/collapse button.
When user navigate using tab/ Shift+ Tab or hover on 'Expand/Collapse' button, tooltip is not appeared.
 
Expected Result:
Tooltip should be defined to expand/collapse button.
When user navigate using tab/ Shift+ Tab or hover on 'Expand/Collapse' button, tooltip should be clearly visible to the user.
 
User Impact:
Users with motor impairments who rely on keyboard will face difficulty if tooltip is not defined for expand/collapse button. 
 
MAS Reference Link: [MAS 1.3.3 – Sensory Characteristics](https://aka.ms/MAS1.3.3)

Trap ID: [1.4 - Uncomprehended Element](https://aka.ms/UncomprehendedElement)

“Have feedback on bugs logged, please tag bug as “A11yRCA” and add your feedback in the comment section.”
